### DISCUSSION TITLE
Adding url parameter for mscertsrv

### DISCUSSION THREAD

**Discussion by ctrought** [grindsa/acme2certifier#186] [Ideas]
Before I submit a PR, as per contributing guide posting this for the maintainers to check. Our mscertsrv web enrolment server hosts two different CAs on the same server, and are served at different paths. For this reason, the hardcoded hostname + paths in the `Certsrv` package such as `"https://{0}/certsrv/certnew.cer".format(self.server)` do not allow us to use this with our web enrolment service which is actually hosted at `https://{0}/certsrv/<other-ca>/certnew.cer".format(self.server)`. 

Since it doesn't seem the [original project is accepting PRs](https://github.com/magnuswatn/certsrv/pulls), I was thinking to only include the changes locally and add a new url parameter in the ca handler config. The existing host option would be preserved, but if the `url` option is supplied in the `acme_srv.cfg` it would use that instead.

Let me know your thoughts, and if you are alright if I submit the proposed PR.

**Comment by grindsa** [comment]
Hi,

your proposal makes sense so feel free to submit a PR.

/G.
